app/src/main/database/migrations/20250730172015_add_indexes.sql

```diff
@@ -0,0 +1,29 @@
+-- migrate:up
+ALTER TABLE items ADD COLUMN source_uuid TEXT GENERATED ALWAYS AS (json_extract(data, '$.source')) VIRTUAL;
+CREATE INDEX idx_items_source_uuid ON items(source_uuid);
+
+ALTER TABLE items ADD COLUMN kind TEXT GENERATED ALWAYS AS (json_extract(data, '$.kind')) VIRTUAL;
+CREATE INDEX idx_items_kind ON items(kind);
+
+ALTER TABLE items ADD COLUMN is_read INTEGER GENERATED ALWAYS AS (json_extract(data, '$.is_read')) VIRTUAL;
+CREATE INDEX idx_items_is_read ON items(is_read);
+
+ALTER TABLE items ADD COLUMN last_updated INTEGER GENERATED ALWAYS AS (json_extract(data, '$.last_updated')) VIRTUAL;
+CREATE INDEX idx_items_last_updated ON items(last_updated);
```

Comment:
NB. Changing `VIRTUAL` to `STORED` would compute these columns on write, at the cost of not being able to add them via `ALTER TABLE`....

Reply:
Good idea. Instead of stacking more migrations on here, I just wiped them all and made a new initial migration with this commit: https://github.com/freedomofpress/securedrop-client/pull/2559/commits/ec09fe8fdd8b981a49bc4afc97f24fe02724f983